src/app/action-playground/action-playground.component.html

```diff
@@ -97,3 +97,11 @@
 
 
 <span dspGnd [gnd]="'(DE-588)119166895'"></span>
+
+<hr>
+
+<p>External links Directive</p>
+
+<a href="https://dasch.swiss/" class="linkDasch">DaSCH website</a><br>
+<a href="https://viaf.org/viaf/22936072\" class="link" id="link2">(VIAF)22936072</a><br>
+<a class="link" href="{{domain}}" class="link" id="link2">{{ label }}</a>
```

Comment:
Does it work also with `[href]="domain"`?

Reply:
Can you add another example like:

`<a href="http://localhost:4200/search">Go to search</a>` to see how it handles hrefs on the same domain.